src/llamafactory/extras/constants.py

```diff
@@ -1957,13 +1957,36 @@ def register_model_group(
             DownloadSource.DEFAULT: "Tele-AI/TeleChat-12B",
             DownloadSource.MODELSCOPE: "TeleAI/TeleChat-12B",
             DownloadSource.OPENMIND: "TeleAI/TeleChat-12B-pt",
+        }
+    },
+    template="telechat",
+)
+
+
+register_model_group(
+    models={
+        "TeleChat2-3B-Chat": {
+            DownloadSource.DEFAULT: "Tele-AI/TeleChat2-3B",
+            DownloadSource.MODELSCOPE: "TeleAI/TeleChat2-3B",
+        },
+        "TeleChat2-7B-Chat": {
```

Comment:
The page is 404 status for the 7B model: https://huggingface.co/Tele-AI/TeleChat2-7B

Reply:
Thank you. We have now opened this link for everyone.